api/policies/v1/policy_validation_test.go

```diff
@@ -263,38 +325,47 @@ func TestValidatePolicyUpdate(t *testing.T) {
 					Resources:   []string{"deployments"},
 				},
 			}}, nil, "default", "protect"),
-			clusterAdmissionPolicyFactory([]admissionregistrationv1.RuleWithOperations{
-				{
-					Operations: []admissionregistrationv1.OperationType{admissionregistrationv1.OperationAll},
-					Rule: admissionregistrationv1.Rule{
-						APIGroups:   []string{"apps"},
-						APIVersions: []string{"v1"},
-						Resources:   []string{"deployments"},
-					},
+			clusterAdmissionPolicyFactory([]admissionregistrationv1.RuleWithOperations{{
+				Operations: []admissionregistrationv1.OperationType{admissionregistrationv1.OperationAll},
+				Rule: admissionregistrationv1.Rule{
+					APIGroups:   []string{"apps"},
+					APIVersions: []string{"v1"},
+					Resources:   []string{"deployments"},
 				},
-				{
-					Operations: []admissionregistrationv1.OperationType{admissionregistrationv1.OperationAll},
-					Rule: admissionregistrationv1.Rule{
-						APIGroups:   []string{""},
-						APIVersions: []string{"v1"},
-						Resources:   []string{"pods"},
-					},
+			}}, nil, "default", "protect"),
+			"",
+		},
+		{
+			"policy mode changed from protect to monitor",
```

Comment:
"random" comment location: we should also add a test case ensuring the transition from monitor to protect is allowed

Reply:
done in https://github.com/kubewarden/kubewarden-controller/pull/870/commits/8895033badc57c4c5868a30fff9df7d98c024ce0